testing/bdd/features/validation/3350-value-types.feature

```diff
@@ -0,0 +1,156 @@
+
+@validation @xsd @syntax @times
+Feature: Value types from 3350
+
+  # SPEC-CONFORMANCE: R93
+  Scenario: Valid colour values
+    Given an xml file <xml_file>
+    When it has tts:color attribute with value <color>
+    Then document is valid
+
+    Examples:
+    | xml_file             | color           |
+    | 3350_value_types.xml | white           |
+    | 3350_value_types.xml | rgb(0,0,0)      |
+    | 3350_value_types.xml | rgba(0,0,0,255) |
+    | 3350_value_types.xml | #000000         |
+    | 3350_value_types.xml | #000000FF       |
+
+  Scenario: Invalid colour values
+    Given an xml file <xml_file>
+    When it has tts:color attribute with value <color>
+    Then document is invalid
+
+    Examples:
+    | xml_file             | color           |
+    | 3350_value_types.xml | pinkish-green   |
+    | 3350_value_types.xml | rgb(0,0,999)    |
+    | 3350_value_types.xml | rgba(0,0)       |
+    | 3350_value_types.xml | rgba(0,256,0,0) |
+    | 3350_value_types.xml | #00MM           |
+    | 3350_value_types.xml | 000000          |
+
+
+  # SPEC-CONFORMANCE: R94
+  Scenario: Valid extent values
+    Given an xml file <xml_file>
+    When it has extent attribute with value <extent>
```

Comment:
This is extent on a `region`, right? Just asking because `tts:extent` is allowed on the `tt` element but in that case _must_ have pixel values. From https://www.w3.org/TR/ttml1/#style-attribute-extent :
> If `tts:extent` is specified on the `tt` element, then the width and height must be expressed in terms of two `<length>` specifications, and these specifications must be expressed as non-percentage, definite lengths using pixel units.